Added info about Timofei Rachkovsky

yaramk opened the pull request:
There are three changes to the site:
1. Eng description of Timofei Rachkovsky 
2. Ru description of Timofei Rachkovsky  
3. Photo of Timofei Rachkovsky

yaramk committed:
Create Timofei Rachkovsky.md

The new description for the website Timofei Rachkovsky

yaramk committed:
Create Timofei Rachkovsky.ru.md

Ru translation to Timofei Rachkovsky

yaramk committed:
Timofei Rachkovsky photo to the site

Timofei Rachkovsky photo to the site

biodranik commented:
Неплохо для начала, но есть нюансы в текстах. Подробнее отпишу позже.

yaramk commented:
Хорошо! Спасибо!

пт, 4 мая 2018, 5:59 PM Alexander Borsuk <notifications@github.com>:

> Неплохо для начала, но есть нюансы в текстах. Подробнее отпишу позже.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/biodranik/landing-hugo/pull/23#issuecomment-386628381>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AlJ5CF5NkpNar6wZOMO6KwrYhtzvvUXHks5tvGy4gaJpZM4TyvIx>
> .
>

biodranik commented on content/team/Timofei Rachkovsky.ru.md:
```diff
@@ -0,0 +1,14 @@
+---
+title: Тимофей Рачковский
+job_title: Design Engineer, магистрант
```
Инженер-проектировщик, магистрант

biodranik commented on content/team/Timofei Rachkovsky.md:
```diff
@@ -0,0 +1,14 @@
+---
+title: Timofei Rachkovsky
+job_title: Design Engineer, Master's Degree student
+description: In VibroBox team Timofei is Design Engineer. He develops and tests hardware and firmware, as well as responsible for the developing signal processing modules in the spectral domain and data acquisition subsystems.
```
As a Design Engineer, Timofei develops and tests our hardware and firmware. He is also responsible for the development of signal processing algorithms in the spectral domain and data acquisition subsystems.

yaramk replied:
Нужно уточнить про замену модулей алгоритмами.

biodranik commented on content/team/Timofei Rachkovsky.md:
```diff
@@ -0,0 +1,14 @@
+title: Timofei Rachkovsky
+job_title: Design Engineer, Master's Degree student
+description: In VibroBox team Timofei is Design Engineer. He develops and tests hardware and firmware, as well as responsible for the developing signal processing modules in the spectral domain and data acquisition subsystems.
+keywords: Timofei Rachkovsky, design engineer, Master's Degree student, VibroBox, Digital Signal Processing, software testing, hardware testing
```
, firmware, spectral domain, data acquisition

biodranik commented on content/team/Timofei Rachkovsky.md:
```diff
@@ -0,0 +1,14 @@
+share:
+  image: img/team/Timofei Rachkovsky.jpg
+---
+In VibroBox team Timofei is Design Engineer. He develops and tests hardware and firmware, as well as responsible for the developing signal processing modules in the spectral domain and data acquisition subsystems. He is responsible for the interface between the software framework and the web part of the project. He works in a close contact with other developers to maximize the effectiveness of th ...
```
```
As a Design Engineer, Timofei develops and tests our hardware and firmware. He is also responsible for:

* development of signal processing algorithms in the spectral domain;
* data acquisition subsystems;
* data processing framework/UI interaction API;
* user manuals for VibroBox products.

Timofei works in a close contact with other developers to maximize the effectiveness of the final product.
```

yaramk replied:
Круто! Только опять про алгоритмы и модели нужно уточнить

yaramk replied:
а что такое "UI interaction API"?
Если это про пользовательский интерфейс, то Тимофей таким не занимается

biodranik commented on content/team/Timofei Rachkovsky.md:
```diff
@@ -0,0 +1,14 @@
+---
+In VibroBox team Timofei is Design Engineer. He develops and tests hardware and firmware, as well as responsible for the developing signal processing modules in the spectral domain and data acquisition subsystems. He is responsible for the interface between the software framework and the web part of the project. He works in a close contact with other developers to maximize the effectiveness of th ...
+
+Timofei graduated from Belarusian State University of Informatics and Radioelectronics with a bachelor’s degree in Radioinformatics engineering. Timofei’s thesis theme is “Data gathering and transmission unit for the condition monitoring systems.”
```
... graduated in 20xx from ...

... theme was "Data collection and transmission unit for condition monitoring systems."

biodranik commented on content/team/Timofei Rachkovsky.md:
```diff
@@ -0,0 +1,14 @@
+
+Timofei graduated from Belarusian State University of Informatics and Radioelectronics with a bachelor’s degree in Radioinformatics engineering. Timofei’s thesis theme is “Data gathering and transmission unit for the condition monitoring systems.”
+
+Now Timofei is a Master’s Degree student at BSUIR in the speciality of Radioengineering, including systems and devices for radionavigation, radar, and television. His thesis theme: “The condition monitoring of mashinery on the basis of packet wavelet transform.”
```
thesis theme is "Condition monitoring of machinery based on a packet wavelet transform."

biodranik commented on content/team/Timofei Rachkovsky.ru.md:
```diff
@@ -0,0 +1,14 @@
+---
+title: Тимофей Рачковский
+job_title: Design Engineer, магистрант
+description: В проекте VibroBox Тимофей работает инженером-проектировщиком. Он разрабатывает и тестирует аппаратное и программное обеспечение, а также ответственен за разработку модулей цифровой обработки сигналов в спектральной области и подсистем сбора данных.
```
....а так же отвечает за разработку....

biodranik commented on content/team/Timofei Rachkovsky.ru.md:
```diff
@@ -0,0 +1,14 @@
+title: Тимофей Рачковский
+job_title: Design Engineer, магистрант
+description: В проекте VibroBox Тимофей работает инженером-проектировщиком. Он разрабатывает и тестирует аппаратное и программное обеспечение, а также ответственен за разработку модулей цифровой обработки сигналов в спектральной области и подсистем сбора данных.
+keywords: Тимофей Рачковский, инженер-проектировщик, магистрант, VibroBox, цифровая обработка сигналов, тестирование программного обеспечения, тестирование аппаратного обеспечения.
```
лучше инженер и проектировщик разбить на два слова и через запятую, это кейворды для поисковиков.

yaramk replied:
Мы с Тимофеем думаем, что это не совсем верно, т.к. получится две отдельные специальности.  Давай не будем менять. В русском яз это устоявшаяся специальность.

biodranik replied:
Здесь кейворды, не имеющие никакого отношения к специальности. Это ключевые слова для поиска в некоторых поисковых системах. Разбивать имеет смысл, чтобы находило по отдельным словам инженер и проектировщик.

biodranik commented on content/team/Timofei Rachkovsky.ru.md:
```diff
@@ -0,0 +1,14 @@
+share:
+  image: img/team/Timofei Rachkovsky.jpg
+---
+В проекте VibroBox Тимофей работает инженером-проектировщиком. Он разрабатывает и тестирует аппаратное и программное обеспечение, а также ответственен за разработку модулей цифровой обработки сигналов в спектральной области и подсистем сбора данных. Он проектирует интерфейсную часть между программной средой разработки и веб-частью проекта. Также Тимофей создает руководства по эксплуатации продукт ...
```
... а так же отвечает за разработку...

biodranik commented on content/team/Timofei Rachkovsky.ru.md:
```diff
@@ -0,0 +1,14 @@
+---
+В проекте VibroBox Тимофей работает инженером-проектировщиком. Он разрабатывает и тестирует аппаратное и программное обеспечение, а также ответственен за разработку модулей цифровой обработки сигналов в спектральной области и подсистем сбора данных. Он проектирует интерфейсную часть между программной средой разработки и веб-частью проекта. Также Тимофей создает руководства по эксплуатации продукт ...
+
+Тимофей получил степень бакалавра по специализации "Радиоинформатика" в Белорусском государственном университете информатики и радиоэлектроники. Защитил диплом по теме "Блок накопления и передачи данных для системы оценки технического состояния оборудования". 
```
после названия универа можно написать в скобках (БГУИР) и в абзаце ниже использовать сокращение.

yaramk committed:
The first update of Timofei Rachkovsky.md

This is the first update of Timofei Rachkovsky.md

yaramk committed:
The 2 update of Timofei Rachkovsky.md

In this update was added: Belarusian State University of Informatics and Radioelectronics (BSUIR)

yaramk committed:
Update Timofei Rachkovsky.md

the word "modules" was changed on "algorithms"

yaramk committed:
Update Timofei Rachkovsky.ru.md

Correction of the text in Timofei Rachkovsky.ru.md
